### DISCUSSION TITLE
Access token not working or field blank

### DISCUSSION THREAD

**Discussion by Sandr-aa** [mircokroon/minecraft-world-downloader#231] [Q&A] [answered]
Like said above, I have the problem that the Access token(s) generated with MultiMC aren't working.
The vanilla launcher just leaves the filed blank.
![image](https://user-images.githubusercontent.com/75202782/137632597-79795f42-abd5-41b5-a9c9-d226c9fa686d.png)
Account Type: Microsoft (migrated)

**Comment by mircokroon** [comment]
Have a look at the Authentication page on the wiki for updated instructions: https://github.com/mircokroon/minecraft-world-downloader/wiki/Authentication

**Reply by Sandr-aa** [reply]
None of the following are working:
C:\>wmic process where 'java%'
C:\>wmic process where 'JAVA%'
C:\>wmic process where 'JAVAW%'
C:\>wmic process where 'javaw%'
C:\>wmic process where 'javaw'
C:\>wmic process where 'JAVAW'
C:\>wmic process where 'JAVA'
C:\>wmic process where 'java'
wmic process where 'JAVA_HOME%'
wmic process where 'JAVA_HOME'
wmic process where 'java_home'
wmic process where 'java_home%'

**Comment by mircokroon** [comment, marked as answer]
Looks like you missed part of the command, I suggest copy-pasting it to avoid mistakes.

**Reply by Sandr-aa** [reply]
using exactly "wmic process where "name like 'java%'"" results in this: [output.txt](https://github.com/mircokroon/minecraft-world-downloader/files/7360198/output.txt)
but it doesn't contain "accessToken"

**Reply by mircokroon** [reply]
Is this after launching the game?

**Reply by Sandr-aa** [reply]
Yes, the game was running (MultiMC) at that time

**Reply by mircokroon** [reply]
Right, I guess this doesn't work correctly with MultiMC then. I'd suggest using the default launcher for now.
